recipe/build.sh

```diff
@@ -21,4 +21,5 @@ fi
 ./configure --with-pic --prefix=$PREFIX --disable-dependency-tracking
 make
 make check
+ln README.md README  # hack; snappy autotools build is broken in 1.1.6
```

Comment:
I see. Thanks. For reference, these [lines]( https://travis-ci.org/conda-forge/snappy-feedstock/builds/255686622#L662-L665 ) show the problem.

So why symlink and not just copy?